rustler_mix/lib/rustler/compiler.ex

```diff
@@ -122,13 +122,47 @@ defmodule Rustler.Compiler do
   defp make_build_mode_flag(args, :release), do: args ++ ["--release"]
   defp make_build_mode_flag(args, :debug), do: args
 
+  defp get_target_os_type(nil) do
+    :os.type()
+  end
+
+  defp get_target_os_type(target) when is_binary(target) do
+    os_type =
+      [
+        %{pattern: ~r/.*-linux.*/, os_type: {:unix, :linux}},
```

Comment:
Should we simply use `:unix` here instead of `{:unix, _}`?

Reply:
I'm not quite sure if I fully understand this comment...

From my perspective, we are sort of replacing the `:os.type/0` function, therefore, we are supposed to return a tuple with 2 elements, right...? And the code that uses `get_target_os_type/1` wants to distinguish between `{:unix, :darwin}` and other *NIX systems, `{:unix, _}`.

Could you please elaborate more on this? Thanks

Reply:
> Could you please elaborate more on this?

Yes, I understand the motivation. I was wondering if it makes sense to deviate from that, as we don't use the second element of the tuple elsewhere (yet?). Lets stick with your approach, there is no important reason to use atoms directly.